R/ui.R

```diff
@@ -100,10 +104,11 @@ style_dir <- function(path = ".",
                       ...,
                       style = tidyverse_style,
                       transformers = style(...),
+                      filetype = "R",
```

Comment:
can we change that to `c("r")` also?

Reply:
Sorry did the other merge from my phone and I thought you'd changed it already. Happy to change this too.

Reply:
@krlmlr do you prefer to stick with `"R"` everywhere instead of `"r"`? Its kind of a philosophical debate I guess...

Reply:
Can we use `"R"` everywhere? It doesn't seem to affect the code, but the default file extension is `.R` and not `.r`, and we should be consistent here.

Reply:
I agree that the default here and elsewhere should be uppercase. I think @lorenzwalthert wanted to emphasize that this argument can be/is a vector so wanted to use c("R") instead. I think the case inconsistency was incidental/accidental.

Reply:
I thought `c("R")` was an artifact... I feel it's enough if we just mention that the argument also accepts vectors in the documentation.

Reply:
ok, fine.